hrjobcontract/js/test/services/utils_test.js

```diff
@@ -0,0 +1,47 @@
+define([
+    'common/angular',
+    'common/angularMocks',
+    'job-contract/app'
+], function (angular, Mock) {
+    'use strict';
+
+    describe('UtilsService', function () {
+        var UtilsService;
+        var API;
+
+        beforeEach(module('hrjc'));
+        beforeEach(inject(function (_UtilsService_, _API_) {
+            UtilsService = _UtilsService_;
+            API = _API_;
+        }));
+
+        beforeEach(function () {
+            spyOn(API, 'resource').and.callFake(function () { return { get: function () {} } })
+            UtilsService.getAbsenceType();
```

Comment:
I think the only thing that should be in this common `beforeEach` should be the `spyOn` bit, all the others are specific to the `describe`s for each method

```js
describe('getAbsenceType', function () {
  beforeEach(function () {
    UtilsService.getAbsenceType();
  });
});

// etc..
```